.github/workflows/acceptance.yml

```diff
@@ -309,10 +309,10 @@ jobs:
         if: needs.files-changed.outputs.backend == 'true'
 
   visual-diff:
-    if: ${{ always() }}
+    if: always()
     # This guarantees that we will only schedule Visual Snapshots if all
     # workflows that generate artifacts succeed
-    needs: [acceptance, frontend, chartcuterie]
+    needs: [acceptance, frontend, chartcuterie, files-changed]
```

Comment:
Because the Diff snapshots step below tries to access the variable: `if: needs.files-changed.outputs.acceptance == 'true'`